src/main/java/com/google/firebase/auth/ListProviderConfigsPage.java

```diff
@@ -261,7 +261,7 @@ public ListSamlProviderConfigsResponse fetch(int maxResults, String pageToken)
 
     ListProviderConfigsPage<T> create() throws FirebaseAuthException {
       ListProviderConfigsResponse<T> batch = source.fetch(maxResults, pageToken);
-      return new ListProviderConfigsPage<T>(batch, source, maxResults);
+      return new ListProviderConfigsPage<>(batch, source, maxResults);
```

Comment:
Should we do a project wide search for similar missing generics? :)

Reply:
We should at some point. Hopefully we can get an IDE to do the work all in one go.